warehouse/snowflake/snowflake.go

```diff
@@ -147,7 +149,20 @@ func checkAndIgnoreAlreadyExistError(err error) bool {
 	return true
 }
 
-func (sf *HandleT) loadTable(tableName string, columnMap map[string]string, accessKeyID, accessKey string) (err error) {
+func (sf *HandleT) authString() string {
+	var auth string
+	switch sf.CloudProvider {
+	case "AWS":
+		auth = fmt.Sprintf(`CREDENTIALS = (AWS_KEY_ID='%s' AWS_SECRET_KEY='%s')`, warehouseutils.GetConfigValue("accessKeyID", sf.Warehouse), warehouseutils.GetConfigValue("accessKey", sf.Warehouse))
```

Comment:
Consider using a string constant instead of using accessKeyID

Reply:
@sivashanmukh changed all configs to be read from constants